### ISSUE TITLE
Images behind proxy

### ISSUE THREAD

**Issue by coenier** [dompdf/dompdf#1039]
This line of code 
https://github.com/dompdf/dompdf/blob/develop/src/Image/Cache.php#L97
should read 

```
$image = file_get_contents($full_url, null, (($dompdf) ? $dompdf->getHttpContext() : null));
```

So that the proxy settings that could be supplied to setHttpContext on the DOMPDF class be used to retrieve the image if the server/script is running behind proxy server.

**Comment by bsweeney** [comment]
Thank you for that! Feel free to submit a PR (if you're inclined) otherwise we'll patch that for the upcoming release.

**Comment by bsweeney** [comment]
This issue was addressed in the develop branch (0.7.x) by #1051.
